documentation/modules/cruise-control/proc-fixing-problems-with-kafkarebalance.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='proc-fixing-problems-with-kafkarebalance-{context}']
+
+= Fixing problems with a `KafkaRebalance` resource
+
+If an issue occurs in the process of creating a `KafkaRebalance` resource, or when interacting with Cruise Control, the error is reported in the resource status, along with details of how to fix it. 
+The resource also moves to the `NotReady` state.
+
+To continue with the cluster rebalance operation, you must fix the problem on the `KafkaRebalance` resource itself. 
+Problems can include the following:
+
+* A misconfigured parameter.
+* The Cruise Control server is not reachable. 
+
+When you believe that you have fixed the issue, you need to add the `refresh` annotation to the resource.
+During a “refresh”, a new optimization proposal is requested from the Cruise Control server.
+
+.Prerequisites
+
+* You have xref:proc-implementing-optimization-proposal-{context}[implemented an optimization proposal].
+
+* The status of the `KafkaRebalance` custom resource for the rebalance operation is `NotReady`.
+
+.Procedure
+
+. Get information about the error from the `KafkaRebalance` status and resolve the issue if possible.
+
+. Annotate the `KafkaRebalance` resource in Kubernetes:
++
+[source,shell,subs="+quotes"]
+----
+kubectl annotate kafkarebalance _cluster-rebalance-name_ strimzi.io/rebalance=refresh
+----
+
+. Check the status of the `KafkaRebalance` resource:
++
+[source,shell,subs="+quotes"]
+----
+kubectl describe kafkarebalance.kafka.strimzi.io -n <namespace> <rebalance-cr-name>
```

Comment:
We should be consistent and use just `kafkarebalance` here

Reply:
Also why are you using namespace here when none of the other command examples specify it? Probably best to remove it and be consistent.